agent/05_agent_configure.sh

```diff
@@ -264,7 +371,20 @@ if [[ ! -z "${MIRROR_IMAGES}" ]]; then
 fi
 
 if [[ "${NUM_MASTERS}" > "1" ]]; then
-   set_api_and_ingress_vip
+  if [[ "${AGENT_PLATFORM_TYPE}" == "none" ]]; then
+    # for platform "none" both API and INGRESS point to the same
+    # load balancer IP address
+    get_vips
```

Comment:
Is it necessary to call get_vips when using SNO (NUM_MASTERS=1)?

Reply:
SNO would go to the else clause, so we are ok here.